test/integration/cds_integration_test.cc

```diff
@@ -195,6 +195,21 @@ TEST_P(CdsIntegrationTest, CdsClusterUpDownUp) {
 TEST_P(CdsIntegrationTest, CdsClusterTeardownWhileConnecting) {
   initialize();
   test_server_->waitForCounterGe("cluster_manager.cluster_added", 1);
+  // With https://github.com/envoyproxy/envoy/pull/23921 ClusterInfo::trafficStats is lazy init.
+  // We need to trigger creation of ClusterInfo::trafficStats() by calling the * operator.
+  absl::Notification n;
+  test_server_->server().dispatcher().post([&]() {
+    (void)*test_server_->server()
```

Comment:
can you break this into two lines? One where you capture the traffic-stats in a temp, and the second where you * the lazy-init temp? I feel like the way you have it it's harder to read what's going on.

Also, in Envoy style we don't put the `(void)` in for this use-case, although IMO personally I think it conveys meaning.